views/registration.html

```diff
@@ -0,0 +1,14 @@
+{{if .Success}}
+    <h1>You have been successfully registered!</h1>
+{{else}}
+<h1>Registration</h1>
+<form method="POST">
+    <label>Email:</label><br />
+    <input type="text" name="email" id="email"><br />
+    <label>Username:</label><br />
+    <input type="text" name="username" id="username"><br />
+    <label>Password:</label><br />
+    <input type="text" name="password" id="password"><br />
```

Comment:
Not an issue right now, because this is only for setup, but the password field should be of type `"password"`.